kernelci/build.py

```diff
@@ -872,3 +900,28 @@ def publish_kernel(kdir, install='_install_', api=None, token=None,
         resp.raise_for_status()
 
     return True
+
+
+def recreate_dir(path):
+    """ Deletes and recreates a directory
+
+    Deletes given directory and all its subdirectiries if extist and creates
+    an empty directory at a given path. If the intermediate-level
+    directories don't exist they will be created.
+
+    *path* is a directory name or a path
+    """
+    if os.path.exists(path):
+        shutil.rmtree(path)
+    os.makedirs(path)
+
+
+def download_file(url, dest_filename, chunk_size=1024):
```

Comment:
This is a helper function used within this module, so please call it `_download_file()`.